sys-cluster/drbd-utils/drbd-utils-9.12.1.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 multilib udev
+
+LICENSE="GPL-2"
+
+DESCRIPTION="mirror/replicate block-devices across a network-connection"
+SRC_URI="http://www.linbit.com/downloads/drbd/utils/${P/_/}.tar.gz"
+HOMEPAGE="https://www.linbit.com/drbd/"
+
+KEYWORDS="~amd64 ~x86"
+IUSE="heartbeat pacemaker +udev xen"
+SLOT="0"
+
+DEPEND="heartbeat? ( sys-cluster/heartbeat )
+	pacemaker? ( sys-cluster/pacemaker )
+	udev? ( virtual/udev )
+	!!sys-cluster/drbd"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${P/_/}"
+
+src_prepare() {
+	# respect LDFLAGS, #453442
+	sed -e "s/\$(CC) -o/\$(CC) \$(LDFLAGS) -o/" \
+		-e "/\$(DESTDIR)\$(localstatedir)\/lock/d" \
+		-i user/*/Makefile.in || die
+
+	# respect multilib
+	sed -i -e "s:/lib/:/$(get_libdir)/:g" \
+		Makefile.in scripts/{Makefile.in,global_common.conf,drbd.conf.example} || die
+	sed -e "s:@prefix@/lib:@prefix@/$(get_libdir):" \
+		-e "s:(DESTDIR)/lib:(DESTDIR)/$(get_libdir):" \
+		-i user/*/Makefile.in || die
+
+	# correct install paths (really correct this time)
+	sed -i -e "s:\$(sysconfdir)/bash_completion.d:$(get_bashcompdir):" \
+		scripts/Makefile.in || die
+
+	# don't participate in user survey bug 360483
+	sed -i -e '/usage-count/ s/yes/no/' scripts/global_common.conf || die
+	sed -i -e "s:\$(sysconfdir)/udev:$(get_udevdir):" scripts/Makefile.in || die
+
+	epatch "${FILESDIR}"/run-lock.patch
```

Comment:
Starting with EAPI=7, epatch is banned and eapply must be used. See https://devmanual.gentoo.org/ebuild-writing/functions/src_prepare/epatch/index.html

Reply:
Better change the patch to work with p1 and define the PATCHES array